README.md

```diff
@@ -35,6 +35,8 @@ We'll organise into different streams here.
 * Weather App
 * Recipe search App
 * E-commerce app
+* Tinder Clone
```

Comment:
Dei LoL! 🤣

Reply:
@danishsshaikh Resolve this conversation once you have seen this! 🤣

Reply:
haha, got it xD